.github/workflows/CI.yml

```
@@ -364,6 +365,7 @@ jobs:
         run: |
             ./build0.sh
             export CXXFLAGS="-Werror"
+            export CFLAGS="-Werror"
             cmake . -GNinja \
               -DCMAKE_BUILD_TYPE=Release \
               -DWITH_LLVM=yes \
```

```suggestion
              -DWITH_LLVM=yes \
              -DCMAKE_VERBOSE_MAKEFILE= yes \
```

I had my trouble seeing which flags are actually used, so this would help.